recipes/gn/all/conandata.yml

```diff
@@ -0,0 +1,7 @@
+sources:
+  "1868":
```

Comment:
Could you give me a pointer where this version is defined?

Reply:
That is deducted from the git commit sha. distance from the last tag I guess. 
Check my patch https://github.com/conan-io/conan-center-index/pull/3668/files#diff-fbfa2e78795701cfca25dd41e7ebed51dcbba399bf32f26e8fe7f16556e00cf2R34

Please note that the build script expect the source code to be in a git repo, however the .tar.gz file does not include git history. So I had to patch the build script and set the version myself.